### ISSUE TITLE
Missing Test Case - 522. Longest Uncommon Subsequence II

### ISSUE THREAD

**Issue by JamesKPolk** [LeetCode-Feedback/LeetCode-Feedback#4624]
#### Your LeetCode username

RunSprly

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [x] Missing Test Cases 

#### Description of the bug

The judge does not accept arrays of size 1, even though that is within the constraints.

The constrains are stated as follows:
1 <= strs.length <= 50
1 <= strs[i].length <= 10
strs[i] consists of lowercase English letters.

When I enter in a size 1 array as a testcase, the judge returns a status
of "Invalid Testcase", and says the expected answer is 
"Line 4: java.lang.AssertionError". 
(This happens regardless of language selected.)

#### Code you used for Submit/Run operation

```
class Solution {
    public int findLUSlength(String[] strs) {
        int n = strs.length;
        
        boolean[] checked = new boolean[n];
        
        int maxLUS=-1;
        
        for(int i=0; i<n; i++){
            if(!checked[i]){
                int len = strs[i].length();
                boolean match=false;
                
                if(len<=maxLUS){
                    continue;
                }
                
                //System.out.println("i = " + i);
                
                for(int j=0; j<n; j++){
                    if(i!=j){
                        int len2 = strs[j].length();
                        if(len==len2){
                            if(strs[i].equals(strs[j])){
                                checked[j]=true;
                                match=true;
                                break;
                            }
                        }else if(len>len2){
                            if(isSubsequence(strs[i],strs[j])){
                                checked[j]=true;
                            }
                        }else if(len2>len){
                            if( isSubsequence(strs[j],strs[i]) ){
                                match=true;
                                break;
                            }
                        }
                    }
                }
                
                if(!match){
                    maxLUS = Math.max(maxLUS,len);
                }
            }
        }
        
        return maxLUS;
    }
    
    //returns true if S is a subsequence of L;
    public boolean isSubsequence(String L, String S){
        int len = L.length();
        int len2 = S.length();
        
        int a=0; //L index
        int b=0; //S index
        
        while(b<len2){
            if(a>=len){
                return false;
            }
            
            if(L.charAt(a) == S.charAt(b)){
                a++;
                b++;
            }else{
                a++;
            }
        }
        
        return true;
    }
}
```

#### Language used for code

Java

#### Expected behavior

For an array of size 1, there would be no other string to compare to,
so the longest uncommon subsequence would be all of strs[0].
I would expect the output to be the length of strs[0].

For the input ["a"], I would expect the output 1.

Alternatively, the first constraint could be altered.

#### Screenshots

<img width="311" alt="constraints" src="https://user-images.githubusercontent.com/12899835/131180204-21f3c943-f358-4659-8727-9a233c21f5b5.PNG">
<img width="618" alt="shouldnt be invalid" src="https://user-images.githubusercontent.com/12899835/131180256-d5c7d7ca-20bc-4557-9b06-f8af76df6a5a.PNG">

**Comment by rezadodge** [comment]
I was going to report this issue, too.

**Comment by ampanasiuk** [comment]
Duplicate of #4620

**Comment by lc-kay** [comment]
Hi @JamesKPolk 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-kay** [comment]
@JamesKPolk We apologize for the delayed notice, but this issue was resolved with #4620. Thank you.
